commit a89a836acc49994e7cc900af2857f43f2a5299ba
Author: Jason Dillaman <dillaman@redhat.com>
Date:   Thu Jan 24 20:07:35 2019 -0500

    mgr/MgrClient: reduce log level when issuing a command when not connected
    
    The mgr client will eventually connect and issue the command so it's not
    necessary to spam the logs at level 0.
    
    Signed-off-by: Jason Dillaman <dillaman@redhat.com>

diff --git a/src/mgr/MgrClient.cc b/src/mgr/MgrClient.cc
index 5a9541d32aa..6a61317aa06 100644
--- a/src/mgr/MgrClient.cc
+++ b/src/mgr/MgrClient.cc
@@ -436,7 +436,7 @@ int MgrClient::start_command(const vector<string>& cmd, const bufferlist& inbl,
     auto m = op.get_message({});
     session->con->send_message2(std::move(m));
   } else {
-    ldout(cct, 0) << "no mgr session (no running mgr daemon?), waiting" << dendl;
+    ldout(cct, 5) << "no mgr session (no running mgr daemon?), waiting" << dendl;
   }
   return 0;
 }
